python/lsst/ap/verify/ingestion.py

```diff
@@ -93,16 +93,15 @@ class DatasetIngestConfig(pexConfig.Config):
         default=9999,
         doc="Calibration validity period (days). Assumed equal for all calib types.")
 
-    textDefectPath = pexConfig.Field(
+    curatedCalibPaths = pexConfig.ListField(
         dtype=str,
-        default=None,
-        optional=True,
-        doc="Path to top level of the defect tree.  This is a directory with a directory per sensor. "
-            "Set to None to disable defect ingestion."
+        default=[],
+        doc="Paths to top level of each curated calib's tree. Each is a "
+            "directory with a directory per sensor."
```

Comment:
Rephrase: "Paths to the top level of each curated calib's tree (e.g., defects, crosstalk). Each path should be a directory which contains one subdirectory per sensor."